src/components/Correction/CompletingParty.vue

```diff
@@ -53,11 +52,7 @@ export default class CompletingParty extends Mixins(CommonMixin) {
 
   /** The original Completing Party if found, otherwise undefined. */
   get originalCompletingParty () : OrgPersonIF {
-    if (this.getOriginalIA) {
-      return this.getCompletingParty(this.getOriginalIA.parties)
-    } else {
-      return this.getCompletingParty(this.getEntitySnapshot?.orgPersons)
-    }
+    return this.getCompletingParty(this.getEntitySnapshot?.orgPersons)
```

Comment:
This is a good example of the kinds changes in this refactor: stop looking at the original (corrected) filing and instead look at the business snapshot (ie, latest) data.

THIS change affects only BEN IA corrections, and I have tested it locally and it works correctly.